docs/cluster_spec.md

```diff
@@ -24,11 +24,15 @@ spec:
 When configuring a LoadBalancer, you can also choose to have a public ELB or an internal (VPC only) ELB.  The `type`
 field should be `Public` or `Internal`.
 
+Additionally, you can increase idle timeout of the load balancer by setting its `idleTimeoutSeconds`. The default idle timeout is 5 minutes, with a maximum of 1200 seconds (20 minutes) being allowed by AWS.
```

Comment:
The AWS maximum is 3600.  http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/config-conn-drain.html